templates/e-commerce/product-details.amp.html

```diff
@@ -116,16 +116,22 @@ <h2 class="h7 block pb1">Reviews</h2>
       <div class="col-12 flex flex-wrap">
         <hr class="xs-hide sm-hide mt4" />
         <div class="col-12 md-col-6 px2 md-pl7 commerce-product-desc">
-          <section class="pt4 pb4 md-pt6 md-px4">
+          <section class="pt3 md-pt6 md-px4">
             <h2 class="h5 md-h4">Overview</h2>
-            <p class="mt2">Id lacus amet. Aliquam eos nunc ut scelerisque lacinia, eu rutrum id, vestibulum aliqua vivamus luctus eu rhoncus
+            <p class="mt2 mb3">Id lacus amet. Aliquam eos nunc ut scelerisque lacinia, eu rutrum id, vestibulum aliqua vivamus luctus eu rhoncus
               ut, sodales id. Velit lacus, fermentum neque et sagittis, ac venenatis volutpat, dolore neque feugiat proin
               fermentum odio, odio arcu in eu wisi. </p>
+              <hr/>
+              <div class="pt3 md-pt4 md-pb4">
+                <h2 class="h5 md-h4 mb2">Reviews</h2>
+                {{> partials/review.snip.html}}
```

Comment:
I think you will need to have review take in an array (list of objects) instead of including it 3 times.